plugins/modules/zos_archive.py

```diff
@@ -0,0 +1,879 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) IBM Corporation 2020, 2022
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: zos_archive
+version_added: "1.7.0"
+author:
+  - Oscar Fernando Flores Garcia (@fernandofloresg)
+short_description: Archive a dataset on z/OS.
+
+description:
+  - Creates or extends an archive.
+  - The source and archive are on the remote host,
+    and the archive is not copied to the local host.
+
+options:
+  path:
+    description:
+      - Remote absolute path, glob, or list of paths or globs for the file or files to compress or archive.
+    type: list
+    required: true
+    elements: str
+  format:
+    description:
+      - The type of compression to use.
+    type: dict
+    required: false
+    suboptions:
+      name:
+        description:
+          - The name of the format to use.
+        type: str
+        required: false
+        default: gz
+        choices:
+          - bz2
+          - gz
+          - tar
+          - zip
+          - terse
+          - xmit
+          - pax
+      format_options:
+        description:
+          - Options specific to each format.
+        type: dict
+        required: false
+        suboptions:
+          terse_pack:
+            description: Pack option to use for terse format.
+            type: str
+            choices:
+              - PACK
+              - SPACK
+          xmit_log_dataset:
+            description: Provide a name of data set to use for xmit log.
+            type: str
+          use_adrdssu:
+            description:
+              - If set to true, after unpacking a data set in C(xmit) or C(terse) format
+                it will perform a single DFSMSdss ADRDSSU DUMP step.
+            type: bool
+            default: False
+  dest:
+    description: The file name of the dest archive.
+    type: str
+    required: false
+  exclude_path:
+    description:
+        - Remote absolute path, glob, or list of paths, globs or data set name patterns for the file, files or data sets to exclude
+           from path list and glob expansion.
+    type: list
+    required: false
+    elements: str
+  group:
+    description:
+      - Name of the group that should own the filesystem object, as would be fed to chown.
+      - When left unspecified, it uses the current group of the current user unless you are root,
```

Comment:
There is a typo ", as would be as would be" , also have a look at what i wrote in the second bullet, maybe that is shorter and says the same thing, otherwise just the typo.


```
  group:
    description:
      - Name of the group that will own the filesystem object, this is passed the system`chown` command.
      - When left unspecified, it uses the current group of the current user unless you are root,
         in which case it can preserve the previous ownership.
      - This option is only applicable if C(dest) is USS, otherwise ignored.
    type: str
    required: false
```

Reply:
done